catalog/ui/catalog-ui-search/src/main/webapp/component/query-custom/query-custom.view.js

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
We should be able to reuse `query-advanced` component here.

Reply:
Done